cfme/tests/automate/test_method.py

```diff
@@ -55,6 +56,8 @@ def test_method_crud(klass):
         location='inline',
         script='$evm.log(:info, ":P")',
     )
+    view = appliance.browser.create_view(ClassDetailsView)
```

Comment:
As per my understanding only to create view you inherited `appliance`. As per me any entity object capable to access `create_view`. Please try this may be it will work `method.create_view(ClassDetailsView)`

Reply:
Thank you :)